automated-comments/csharp/two-fer/use_null_coalescing_operator_not_is_null_or_empty.md

```diff
@@ -0,0 +1 @@
+You are currently using `string.IsNullOrEmpty` to determine if the default name should be used. As an alternative, you could try using the [null-coalescing operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator) to simplify your code and consider the difference in behaviour.
```

Comment:
Removing "you":

```
Rather than using `string.IsNullOrEmpty` to determine if the default name should be used, consider trying the [null-coalescing operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator) to simplify the code and consider the difference in behaviour.```